zapier/authentication.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  type: 'custom',
+  test: {
+    removeMissingValuesFrom: { body: false, params: false },
+    url: '{{process.env.BASE_URL}}/api/zapier/validate',
```

Comment:
If we move the `external` API to `external` prefix let's not forget to update the conf here